packages/driver/src/cy/commands/files.ts

```diff
@@ -25,7 +25,7 @@ export default (Commands, Cypress, cy, state) => {
       // privileged commands need to send any and all args, even if not part
       // of their API, so they can be compared to the args collected when the
       // command is invoked
-      const userArgs = trimUserArgs([file, encoding, _.isObject(userOptions) ? { ...userOptions } : undefined, ...extras])
+      const userArgs = [file, encoding, _.isObject(userOptions) ? { ...userOptions } : undefined, ...extras]
```

Comment:
Trimming (and hashing) the user args has been moved into the `runPrivilegedCommand` function.